src/openseadragon.js

```diff
@@ -206,6 +206,9 @@
   * @property {Number} [opacity=1]
   *     Opacity of the drawer (1=opaque, 0=transparent)
   *
+  * @property {String|Function} [placeholderFillStyle=null]
```

Comment:
Thanks, i really never used JSDoc before except for OpenSeadragon, i think i'll start reading the docs for it :)